bin/compile

```diff
@@ -123,24 +126,17 @@ if [ -x .flutter/bin/flutter ]; then
   cp -R $SOURCE_DIR/.pub-cache $CACHE_DIR/.pub-cache
 fi
 
+# Move built website inside deploy dir before cleaning files
+mv $SOURCE_DIR/build/web/* $DEPLOY_DIR
+
 # Check  FLUTTER_CLEANUP var to delete all files or keep it.
 if [ "$FLUTTER_CLEANUP" != false ]; then
   print "Cleaning up Flutter source files and the SDK"
 
-  mkdir -p $SOURCE_DIR/TO_DELETE
-
-  mv !("TO_DELETE") TO_DELETE
-  mv TO_DELETE/build/web/* $DEPLOY_DIR
-
-  rm -rf TO_DELETE
-fi
+  rm -rf $SOURCE_DIR 
```

Comment:
This way you only delete the original source files but not the Flutter/Dart artifacts (including the entire SDK and all installed packages). The latter is usually much more important to clean, because it's huge, especially for projects with a lot of dependencies.

This would cause deploys to be very slow and in many cases hit [the slug limit](https://devcenter.heroku.com/articles/slug-compiler#slug-size).